packages/medusa-cli/src/commands/new.js

```diff
@@ -494,18 +535,32 @@ export const newStarter = async args => {
     dbHost,
   } = args
 
-  const dbCredentials = {
+  const dbCredentials = removeUndefined({
     user: dbUser,
     database: dbDatabase,
     password: dbPass,
     port: dbPort,
     host: dbHost,
-  }
+  })
 
-  const { starterPath, rootPath } = await getPaths(starter, root)
+  const { starterPath, rootPath, selectedOtherStarter } = await getPaths(
+    starter,
+    root
+  )
 
   const urlObject = url.parse(rootPath)
 
+  if (selectedOtherStarter) {
+    reporter.info(
+      `Find the url of the Medusa starter you wish to create and run:
+
+medusa new ${rootPath} [url-to-starter]
+
+`
+    )
+    return
+  }
+
   if (urlObject.protocol && urlObject.host) {
     const isStarterAUrl =
       starter && !url.parse(starter).hostname && !url.parse(starter).protocol
```

Comment:
_Comment refers to line 569-604_

Should we include the error map like Gatsby now to use this type of reporting? Otherwise, I think we should delete these and add own our type of clever reporting some time in the future

Reply:
Have updated with a panic handler now that formats errors based on the provided `id`

Reply:
Very nice - I like it :)